src/box/lua/feedback_daemon.lua

```diff
@@ -7,15 +7,24 @@ local http  = require('http.client')
 local fio = require('fio')
 
 local PREFIX = "feedback_daemon"
+local METRICS_PREFIX = "metrics_collector"
```

Comment:
I would add a separate patch adding some prefix, e.g., `feedback` to `PREFIX`, `fiber`, etc., because now it becomes unclear what they refer to.

Reply:
Could you elaborate, please? Why do I need a separate patch? It is just name of the fiber for logs.

Reply:
After this patch we will the daemon will have fields like `fiber` and `metrics_collect_fiber` — I suggest you add the prefix `feedback` to the latter ones for clarity.

Reply:
I've added `send` prefix to some fields in a separate commit.